jquery.pnotify/index.d.ts

```diff
@@ -257,6 +257,36 @@ interface PNotifyOptions {
     labels?: PNotifyLabel;
 }
 
+interface PNotifyStyling {
+    container?: string,
+    notice?: string,
+    notice_icon?: string,
+    info?: string,
+    info_icon?: string,
+    success?: string,
+    success_icon?: string,
+    error?: string,
+    error_icon?: string,
+    // buttons
+    closer?: string,
```

Comment:
The comments indicate the file name pnotify.<comment>.js.
Here's an example for buttons: https://github.com/sciactive/pnotify/blob/master/src/pnotify.buttons.js#L171